README.md

```diff
@@ -394,6 +397,20 @@ git commit --allow-empty -m 'Redeploying'
 git push heroku master
 ```
 
+### Deployment Health Checklist
+
+###### Overview
+We recommend completing our Deployment Health Checklist to help you improve the stability and reliability of your Retool deployment.
+
+Please fill out the checklist and share it with our team. This information will help us better understand your infrastructure so that we can support you through product changes, proactive outreach, and more informed support.
+
+###### Instructions
+Make a copy of the [Deployment Health Checklist](https://docs.google.com/spreadsheets/d/19XYpWTnYrvsllTuM2VQGFWLiXzMHmTNAzZKyWY-sfSU) for your Retool deployment. Add your company name to the document title for reference.
```

Comment:
Not for this PR but we should try and figure out a way to provide a checklist that's not a Google sheet. Not sure who maintains that doc.

Reply:
Totally agree, i'm actually not sure who maintains this either.